encoding/legacy-mb-schinese/gbk/gbk-decode-chinese.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+<head>
+<meta charset="utf-8"/>
+<title>chinese decoding</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<link rel="author" title="Richard Ishida" href="mailto:ishida@w3.org">
+<link rel="help" href="https://encoding.spec.whatwg.org/#names-and-labels">
+<meta name="assert" content="The browser produces the same decoding behavior for a document labeled 'chinese' as for a document labeled 'gbk'.">
+<style>
+ iframe { display:none }
+</style>
+<script src="gbk_index.js"></script>
+<script src="gbk-decoder.js"></script>
+</head>
+
+<body onload="showNodes();">
+
+<iframe src="gbk_chars-chinese.html" name="scriptWindow" id="scrwin"></iframe>
+
+<p class="instructions">Test passes if the number of failures is the same as for test gbk-decode.html</p>
+<div id="log"></div>
+
+<script>
+var tests = []
+
+function iframeRef( frameRef ) {
+    return frameRef.contentWindow
+        ? frameRef.contentWindow.document
+        : frameRef.contentDocument
+	}
+
+function showNodes () {
+	var iframe = iframeRef( document.getElementById('scrwin') );
+	nodes = iframe.querySelectorAll('span')
+	
+	for (var i=0; i<nodes.length; i++) {
+		tests[i] = async_test("U+"+nodes[i].dataset.cp+' '+String.fromCodePoint(parseInt(nodes[i].dataset.cp,16))+' '+gbDecoder(nodes[i].dataset.bytes)+' '+nodes[i].dataset.bytes) 
+		}	
+		
+	for (var i=0; i<nodes.length; i++) {
+		tests[i].step(function() {assert_equals(nodes[i].textContent, gbDecoder(nodes[i].dataset.bytes)) } )
+		tests[i].done()
+			}
+	}
+
+</script>
+<p style="font-size: 70%;">This test takes a while to run. You can raise issues at <a target="_blank" href="https://github.com/w3c/encodingtests/issues">https://github.com/w3c/encodingtests/issues</a></p>
```

Comment:
From a WPT perspective, i agree. But we sometimes access these tests individually from https://www.w3.org/International/tests/repo/results/encoding-dbl-byte.en, where it can be useful.  I'd prefer not to have to maintain different files in WPT and elsewhere, so is it a problem to leave it in?